server-embedded/src/main/java/org/exoplatform/chat/server/ChatServer.java

```diff
@@ -842,6 +842,33 @@ public ChatServer()
     return Response.ok("Updated.");
   }
 
+  @Resource
+  @Route("/isRoomEnabled")
+  public Response.Content isRoomEnabled(String user, String token, String spaceId) {
+    if (!tokenService.hasUserWithToken(user, token))
+    {
+      return Response.notFound("Petit malin !");
+    }
+
+    String room = ChatUtils.getRoomId(spaceId);
+    Boolean isEnabled = chatService.isRoomEnabled(room);
+
+    return Response.ok(isEnabled.toString());
+  }
+
+  @Resource
+  @Route("/updateRoomEnabled")
+  public Response.Content setRoomEnabled(String user, String token, String spaceId, Boolean enabled) {
```

Comment:
This action should restricted to space owner, space managers and spaces administrators IMO

Reply:
Okay will fix it